src/components/routerView.ts

```diff
@@ -35,6 +35,10 @@ export default defineComponent({
         return h(rejection.value.component)
       }
 
+      if (slots.default) {
+        return slots.default({ component, route: router.route })
+      }
```

Comment:
What are you thinking here?

Reply:
Following the pattern vue-router uses. Gives developers a chance to put "middleware" components between router-view and the end component.

Reply:
Sure 🤷